pages/dashboard/index.js

```diff
@@ -78,6 +84,33 @@ export const getServerSideProps = async ({ req }) => {
   return {
     props: {
       session: JSON.stringify(newSession) ?? '', // otherwise nextjs throws error - can't serialize data
+      friendData: [
+        {
+          name: 'Yi Nan Zhang',
+          status: 'In Study Session',
+          image: '',
+        },
+        {
+          name: 'Charles Liu',
+          status: 'In Study Session',
+          image: '',
+        },
+        {
+          name: 'Jimmy Yang',
+          status: 'In Study Session',
+          image: '',
+        },
+        {
+          name: 'Mabel Kwok',
+          status: 'In Study Session',
+          image: '',
+        },
+        {
+          name: 'Eden Chan',
+          status: 'In Study Session',
+          image: '',
+        },
+      ],
```

Comment:
if this is test data, it's better to put thus in something like `
Dashbaord.defaultProps = {
  friendData: <data>,
}
`